### ISSUE TITLE
Google Pay not showing in the payment setting

### ISSUE THREAD

**Issue by MMEurope** [mollie/OpenCart#328]
Google Pay was added as payment method. Please update the module to reflect it.

**Comment by MMEurope** [comment]
We're waiting 3 weeks for a response

**Comment by asotela3911** [comment]
Hi @MMEurope,

Thank you for your patience. Google Pay is on our roadmap for next year, but we don’t have an exact ETA yet. We’ll update you as soon as we have more details.

**Comment by MMEurope** [comment]
It would really be nice if you could please push this. We've been waiting for Google Pay from mollie for a long time. It's finally here. Please prioritise this.

**Comment by MMEurope** [comment]
Guys why was Google Pay not implemented in the latest release? This is a big update for merchants!

**Comment by asotela3911** [comment]
Google Pay was not implemented in the last release. It is still in the closed Beta phase and not yet ready for implementation in OpenCart. We will keep you posted as soon as we have more updates.

**Comment by MMEurope** [comment]
Thank you for your response. Are there any updates on this?

**Comment by asotela3911** [comment]
No updates yet. For the time being, Google Pay is only offered via the HPP.

**Comment by MMEurope** [comment]
We've been waiting almost a year now.

**Comment by QualityWorks** [comment]
Yes, still no Google Pay on the roadmap, I will close this issue for now.
